includes/documents/class-wcpdf-bulk-document.php

```diff
@@ -117,31 +117,31 @@ public function get_pdf() {
 	}
 
 	public function get_html() {
-		// temporarily apply filters that need to be removed again after the html is generated
-		$html_filters = apply_filters( 'wpo_wcpdf_html_filters', array(), $this );
-		$this->add_filters( $html_filters );
-
-		do_action( 'wpo_wcpdf_before_html', $this->get_type(), $this );
```

Comment:
I don't think we can simply remove this here because it is also called in `get_html()`, because the filter here let's you target the whole bulk document, and the filter in `get_html()` lets you target the single document. The `$this` variable returns something different for this filter than the filter within `get_html()`.